packages/demo/src/components/cards/flow-account-card.tsx

```diff
@@ -0,0 +1,196 @@
+import {useFlowAccount} from "@onflow/kit"
+import {useState} from "react"
+
+export function FlowAccountCard() {
+  const [address, setAddress] = useState("0x1654653399040a61") // Flow token contract address
```

Comment:
Which network is this for? Don't we need one per network?

Reply:
Made all contracts addresses network based!